src/components/homeStyle/HomeLayout.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "styled-components";
+import Navbar from "./Navbar";
+
+type Children = {
```

Comment:
reactのpropsなので Props にしましょう！
```suggestion
type Props = {
```